modules/vulnerabilities/unix/http/linuxki_rce/manifests/configure.pp

```diff
@@ -0,0 +1,18 @@
+# Class: linuxki_rce::configure
+# LinuxKI configuration
+#
+class linuxki_rce::configure {
+  $leaked_filenames = ['flagtest'] ##$secgen_parameters['leaked_filenames']
```

Comment:
You need to uncomment out the part where the secgen parameters are used:

  $secgen_parameters = secgen_functions::get_parameters($::base64_inputs_file)
  $leaked_filenames = $secgen_parameters['leaked_filenames']
  $strings_to_leak = $secgen_parameters['strings_to_leak']